cmd/process-agent/subcommands/check/check.go

```diff
@@ -156,7 +164,7 @@ func runCheckCmd(cliParams *cliParams) error {
 		if err := checks.Process.Init(nil, hostInfo); err != nil {
 			return err
 		}
-		checks.Process.Run(0) //nolint:errcheck
+		checks.Process.Run(nextGroupID(), nil) //nolint:errcheck
```

Comment:
Why not just explicitly ignore the error instead of doing `//nolint:errorcheck`? Not a blocker just curious about your thoughts here

Reply:
I forget why this happened here, but I definitely agree `_ = checks.Process.Run(..)` reads better. I will update it here.